omigami/plotting.py

```diff
@@ -1,60 +1,153 @@
+from typing import List
+
 from PIL.PngImagePlugin import PngImageFile
-from pandas import DataFrame
+import pandas as pd
 from rdkit import Chem
 from rdkit.Chem import Draw
+from rdkit.Chem.rdchem import Mol
+import itertools
+import matplotlib.pyplot as plt
+import requests
+
+
+class MandatoryColumnMissingError(Exception):
+    pass
+
+
+# TODO: Dont know about the name though
+class MSPlots:
```

Comment:
ah you turned into a class after all.. then the notebooks are not updated.
and about the naming.. what does the `MS` mean?

Reply:
Mass Spectra. Good point forgot to update them